src/ui/pages/Overview/Positions/NetworkBalance.tsx

```diff
@@ -155,10 +159,12 @@ export function NetworkBalance({
 
   const testnetMode = preferences?.testnetMode?.on;
   const networksPredicate = useMemo(() => {
-    return testnetMode
+    return standard === 'solana'
+      ? (network: NetworkConfig) => network.id.toLowerCase().includes('solana')
```

Comment:
Isn't there a helper for this?

Reply:
This is a temporary way to check for Solana. Later, backend should update the `standard` property of `NetworkConfig`